spacy_llm/tasks/translation/registry.py

```diff
@@ -0,0 +1,59 @@
+from typing import Optional, Type
+
+from ...registry import registry
+from ...ty import ExamplesConfigType, FewshotExample, ShardMapper, ShardReducer
+from ...ty import TaskResponseParser
+from ..util.sharding import make_shard_mapper
+from .parser import parse_responses_v1
+from .task import DEFAULT_TRANSLATION_TEMPLATE_V1, TranslationTask
+from .util import TranslationExample, reduce_shards_to_doc
+
+
+@registry.llm_misc("spacy.TranslationShardReducer.v1")
+def make_shard_reducer() -> ShardReducer:
+    return reduce_shards_to_doc
+
+
+@registry.llm_tasks("spacy.Translation.v1")
+def make_translation_task(
+    target_lang: str,
+    source_lang: Optional[str] = None,
+    template: str = DEFAULT_TRANSLATION_TEMPLATE_V1,
+    parse_responses: Optional[TaskResponseParser[TranslationTask]] = None,
+    prompt_example_type: Optional[Type[FewshotExample]] = None,
+    examples: ExamplesConfigType = None,
+    shard_mapper: Optional[ShardMapper] = None,
+    shard_reducer: Optional[ShardReducer] = None,
+    field: str = "translation",
```

Comment:
I wonder whether we should make the default depending on the target language, basically making it easier for users to have multiple translation components for different target languages. Though I suppose in that case they can just set the field themselves.

Reply:
You mean naming the field `"translation_{self._target_lang}"` by default?

Reply:
Yes, something like that. You'd have to adjust the default value in the function body though. So it's probably not worth the hassle?

Reply:
Probably not. Alternative suggestion: a dict with language names. That would allow easier/automatic merging for multiple translation components with different languages. I. e. instead of `self._translation = ...` we'd have 
```python
self._translation = {
    self._target_lang: ...
}
```
What do you think? Otherwise I'd stick with the current solution.

Reply:
I think that'll make things too complex for users wanting just a single translation. Let's stick with the current implementation :-)